QSB/TransformSync/PlayerHUDMarker.cs

```diff
@@ -14,13 +14,12 @@ class PlayerHUDMarker : HUDDistanceMarker
 
         protected override void InitCanvasMarker()
         {
-            _markerRadius = 0.3f;
+            _markerRadius = 2f;
 
             _markerTarget = new GameObject().transform;
             _markerTarget.parent = transform;
-            // I'm not really sure why this has to be 20 instead of 2 (the player height in units).
-            // But that's the only way it looks right.
-            _markerTarget.localPosition = Vector3.up * 20;
+
+            _markerTarget.localPosition = Vector3.zero;
```

Comment:
Oh. This makes it show above the player's head? Makes sense 😅 That's why it worked so weird.